pyproject.toml

```diff
@@ -18,6 +18,7 @@ exclude_lines = [
 ignore_errors = true
 omit = [
   "tests/*",
+  "nbqa/find_root.py"  # This is taken directly from `black` so I'll trust it
```

Comment:
If we had unittests for `find_project_root`, i.e. with a dummy tempdir structure, we wouldn't need to exclude `find_root.py`, to have 100% test coverage.

Reply:
That's a good point, thanks for not letting me take the lazy way out :) This one will take a little more work, but it'll be worth it